code/_compile_options.dm

```diff
@@ -76,6 +76,13 @@
 #define TESTING
 #endif
 
+//#if defined(UNIT_TESTS)
+//Hard del testing defines
+//#define REFERENCE_TRACKING
+//#define REFERENCE_TRACKING_DEBUG
+//#define FIND_REF_NO_CHECK_TICK
+//#endif
```

Comment:
?

Reply:
These defines enable a proc that calls functions for the unit test, but said proc is not present in our code. I could try to port that wasent sure if we should or not